src/plugins/workflow_designer/src/WorkflowViewController.h

```diff
@@ -88,7 +88,12 @@ class WorkflowScene : public QGraphicsScene {
         return lastMousePressPoint;
     }
 
-    QList<Actor*> getSelectedActors() const;
+    enum ActorsSelector {
+        Selected,
+        All
+    };
+
+    QList<Actor*> getActors(ActorsSelector) const;
```

Comment:
1. Use const T& for non native types
2. Add parameter variable name

Reply:
`const QList<Actor*>&` will be always retunred as empty list

Reply:
Right, I meant the parameter, not the return value.